assets/scss/templates/_guidelines.scss

```diff
@@ -141,51 +142,75 @@
 }
 
 .buttons-card {
-  height: 235px;
   border-radius: 5px;
-  overflow: hidden;
   margin-bottom: 30px;
-  width: 300px;
+  width: 25%;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
   text-align: left;
+
+  &:before {
+    opacity: 0;
+    border-radius: 3px;
+    box-shadow: 0 4px 20px rgba(black, 0.1);
+    transform: scale(1.3, 1.3);
+  }
+
+  &:hover:before {
+    opacity: 1;
+    transform: scale(1, 1);
+  }
+
+  &:after {
+    transition: all 0.3s;
+    border: 1px solid rgba(153, 153, 153, 0.9);
+  }
+
+  &:hover:after {
+    transform: scale(0, 0);
+    opacity: 0;
+  }
+}
+
+.box-button {
+  position: relative;
+  min-height: 260px;
+  transition: all 0.3s;
+}
+
+.box-button::before,
+.btn::after {
+  content: '';
+  position: absolute;
+  transition: all 0.3s;
+  top: -15px;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
+
+.description-button {
```

Comment:
maybe the name is button-description . What do you think?

Reply:
Tenes razon Fede ya lo cambio!